UPGRADING.INTERNALS

```diff
@@ -17,6 +17,11 @@ PHP 8.3 INTERNALS UPGRADE NOTES
 * zend_class_entry now possesses a default_object_handlers field, which
   provides a default object handler when create_object() is not overriding it.
 
+* The following FCI APIs have been removed as they were confusing:
+  - zend_fcall_info_call()
```

Comment:
You could also list zend_fcall_info_args_save, zend_fcall_info_args_restore here

Reply:
Somehow this got lost in the rebase, fixed.